### PR TITLE
Fixes log alert query not saving

### PR DESCRIPTION
**Pull request by anthonygregis**
## Summary

This resolves an issue when defining a log alert query. When a user inputs a log query the value was being set into a local variable to update the histogram but was not updating the formStore that was being used to populate the mutation input variables. This PR should resolve this issue and allow for query to be sent along to the mutation.

Related Issue: #6630 

## How did you test this change?

Test both locally and in self deployed version

## Are there any deployment considerations?

None, purely frontend changes

## Does this work require review from our design team?

No

### CODE REVIEW

**Review by SpennyNDaJets — APPROVED** [review]
Code and click test look good to me!

**Comment by Vadman97** [comment]
@anthonygregis linter seems to be failing
```
[warn] frontend/src/pages/Alerts/LogAlert/LogAlertPage.tsx
[warn] Code style issues found in the above file. Forgot to run Prettier?
```
could you run `yarn format:all`?

**Comment by anthonygregis** [comment]
Sorry about that, will run and push up

**Comment by Vadman97** [comment]
@anthonygregis no worries. looks like the yarn.lock changes are local to your project, do you mind removing that from your PR?

File: yarn.lock [hunk 1]
```diff
@@ -24879,8 +24879,8 @@ __metadata:
 
 "ansi-color@file:./forks/ansi-color::locator=highlight%40workspace%3A.":
   version: 0.0.0
-  resolution: "ansi-color@file:./forks/ansi-color#./forks/ansi-color::hash=291f6b&locator=highlight%40workspace%3A."
-  checksum: ebc4eb866ee788f9088fd981ccec71a0aca50f8b7f798a6ab31e628b917e4554f6bc5f4c9f8119622901e59350b9bead56c69076d8b09af812c3e9111a10547a
```
Comment (SpennyNDaJets) [inline yarn.lock:24883]: @anthonygregis Is this related? Can we revert this change if not, since it looks like it is causing an error in the build

**Comment by anthonygregis** [comment]
@Vadman97 @SpennyNDaJets Will fix yarn.lock issue now, most likely happened because I'm on Windows.
